dev/maven-plugin/src/main/maven/plugin.xml

```diff
@@ -409,6 +409,15 @@
                 <parameter>
                     <name>cassandraEnabled</name>
                 </parameter>
+                <parameter>
+                    <name>kafkaPort</name>
+                </parameter>
+                <parameter>
+                    <name>kafkaAddress</name>
+                </parameter>
```

Comment:
This is not an introduced property, this change simply allows the existing property to be passed to the start service locator mojo, since the service locator now needs it.  For end users, nothing changes, if they defined a `kafkaAddress` in their root pom (as is documented that they can do), it will just work.  The relationship between `kafkaPort` and `kafkaAddress` is a bit odd, but this isn't the PR to address that.